pom.xml

```diff
@@ -147,6 +147,7 @@
 						</goals>
 						<phase>initialize</phase>
 						<configuration>
+							<dotGitDirectory>${project.basedir}/.git</dotGitDirectory>
```

Comment:
I didn't modify this and it worked anyway (after copying the .git folder with the Dockerfile), is this necessary?

Reply:
There is a slight difference between our files, I will use yours so I can revert this change